src/vue/backend/src/templates/listingsCard.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+    <checkbox-label
+        :option="option"
+        @checked="$emit('listingSelected', [$event.data, property.listingID])"
```

Comment:
Events should be kebab-case per Vue docs: https://vuejs.org/v2/guide/components-custom-events.html#Event-Names

Reply:
Where does it say event names should be kebab-case? It simply states that the emitted event name should match the listener identically, noting that listeners are auto transformed to lowercase due to case insensitivity.